test/integration/006_simple_dependency_tests/local_dependency/models/schema.yml

```diff
@@ -4,7 +4,6 @@ sources:
     schema: invalid_schema
     tables:
       - name: my_table
-sources:
```

Comment:
@emmyoop surprisingly fixing the duplicate top level keys in this yaml file in the legacy tests causes it to pass so now really curious what this was doing previously... makes me wonder if we had left 2 `sources` keys in this file by mistake the last time around but test was passing due to pyaml default of only reading the last `source` key of this file.

Reply:
I'm not seeing anything in the tests that indicate we did it on purpose so you may be right.  These tests are going to be rewritten and we'll work through some of the weirdness in them as we go.